experimentation/Diabetes Ridge Regression Scoring.ipynb

```diff
@@ -0,0 +1,123 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Score Data with a Ridge Regression Model Trained on the Diabetes Dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook loads the model trained in the Diabetes Ridge Regression Training notebook, prepares the data, and scores the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import numpy\n",
+    "from azureml.core.model import Model\n",
+    "import joblib"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_path = Model.get_model_path(model_name=\"sklearn_regression_model.pkl\")\n",
+    "model = joblib.load(model_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "raw_data = '{\"data\":[[1,2,3,4,5,6,7,8,9,10],[10,9,8,7,6,5,4,3,2,1]]}'\n",
+    "\n",
+    "data = json.loads(raw_data)[\"data\"]\n",
+    "data = numpy.array(data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Score Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"RequestId\":\"\", \"TraceParent\":\"\", \"NumberOfPredictions\":2}\n",
+      "Test result:  {'result': [5113.099642122813, 3713.6329271385353]}\n"
+     ]
+    }
+   ],
+   "source": [
+    "request_headers = {}\n",
+    "\n",
+    "result = model.predict(data)\n",
+    "print(('{{\"RequestId\":\"{0}\", '\n",
```

Comment:
Because we are directly testing model, can we please ignore printing request headers ?